Explorer/Assets/DCL/PluginSystem/Global/LandscapePlugin.cs

```diff
@@ -52,7 +55,8 @@ public async UniTask InitializeAsync(LandscapeSettings settings, CancellationTok
             emptyParcels = parcelData.Value.GetEmptyParcels();
             ownedParcels = parcelData.Value.GetOwnedParcels();
 
-            terrainGenerator = new TerrainGenerator(landscapeData.Value.terrainData, ref emptyParcels, ref ownedParcels);
+            TerrainGenerator = new TerrainGenerator(landscapeData.Value.terrainData, ref emptyParcels, ref ownedParcels);
+            WorldTerrainGenerator = new WorldTerrainGenerator(landscapeData.Value.worldsTerrainData);
```

Comment:
To add to @mikhail-dcl comments, we could inject these dependencies since they will be used by other plugins to control when the landscape should be built and disposed of.